roles/setup/tasks/main.yml

```diff
@@ -21,6 +21,46 @@
   tags:
     - initial
 
+- name: create data_dir
+  become: yes
+  file:
+    path: /var/lib/validator
+    state: directory
+    owner: "{{ user['name'] }}"
+    group: "{{ user['name'] }}"
+
+- name: data source path
+  command: >-
+    {{ user['home'] }}/venv/bin/python -c "import pkg_resources as pr; print(pr.resource_filename('validator', 'ml/data'))"
```

Comment:
Ah, that's how to get around needing three levels of quoting: the continuation is sufficient to not make the leading {{ error out